src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -551,8 +540,8 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
 
         List<String> studentList = new ArrayList<>();
         if (!repositoryExportOptions.isExportAllStudents()) {
-            studentIds = studentIds.replaceAll(" ", "");
-            studentList = Arrays.asList(studentIds.split("\\s*,\\s*"));
+            studentIds = studentIds.replaceAll("\\s", "");
```

Comment:
According to https://stackoverflow.com/questions/15633228/how-to-remove-all-white-spaces-in-java/36444332 the following version is better

Reply:
```suggestion
            studentIds = studentIds.replaceAll("\\s+", "");
```